traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -144,6 +144,13 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		/**
 		 * 3.x API
 		 */
+		////DeliveryServices
+		{api.Version{3, 1}, http.MethodGet, `deliveryservices/?$`, api.ReadHandler(&deliveryservice.TODeliveryService{}), auth.PrivLevelReadOnly, Authenticated, nil, 22383173943, noPerlBypass},
```

Comment:
We should only need 3.1 routes for POST and PUT. The GET handler should have the minor-version specific handling, and DELETE is version-agnostic.